klayout/drc/rule_decks/sab.drc

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
```

Comment:
2023 @FaragElsayed2

Reply:
@atorkmabrains

I will do that for all DRC files in a separate PR after adding all tables.

Refer to #75